config/config.yaml

```diff
@@ -31,3 +31,13 @@ project_settings:
 
   # default_restoration_project_id - project into which data will be restored by default during restoration process
   # default_restoration_project_id: 'default-restoration-storage-project-id'
+
+retention_settings:
+
+  # young_old_generation_threshold_in_months - for all backups older than this number of months,
+  # retention process will delete all backups except the most recent one.
+  # This doesn't affect backups younger than this threshold
+  young_old_generation_threshold_in_months: 7
+
+  # grace_period_after_source_table_deletion_in_months - number of months since deletion of source table after retention will remove last backup for given table
+  grace_period_after_source_table_deletion_in_months: 7
```

Comment:
I have an idea about splitting both of this parameters into **backup_soft_expiration** and **backup_hard_expiration**. Would it be more intuitive?